# Request
uint8 channel
uint8 dutyCycle
---
# Result
bool success
---
# No Feedback